### PR TITLE
Update changelog for 0.21.0-insiders2.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
 # C/C++ for Visual Studio Code Change Log
 
-* Add refresh button to process picker. [#2885](https://github.com/Microsoft/vscode-cpptools/issues/2885)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:3]: This moved to the bottom.

**Comment by sean-mcmanus** [comment]
@WardenGnaw Wasn't there some other debugger fixes with MIEngine?

**Comment by sean-mcmanus** [comment]
@bobbrow You had some custom configuration provider fix -- let me know if you want to add a line for that. I wasn't sure of the details.

**Comment by WardenGnaw** [comment]
Fixed:
`Extension throwing NullReference error on VS Code Insiders (#2858)` fixed in https://github.com/Microsoft/MIEngine/pull/810

Merging master into the release branch and going to kick off a build and update MIEngine version for the cpptools build.

**Comment by bobbrow** [comment]
The custom config provider issue was a race condition between the provider and IntelliSense resulting in incorrectly configured includes/defines for a file.  I don't think a GitHub issue was opened for it.

**Review by bobbrow — APPROVED** [review]

**Review by grdowns — APPROVED** [review]

**Pull request merged**